staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3837,7 +3837,7 @@ type SeccompProfile struct {
 	// localhostProfile indicates a profile defined in a file on the node should be used.
 	// The profile must be preconfigured on the node to work.
 	// Must be a descending path, relative to the kubelet's configured seccomp profile location.
-	// Must only be set if type is "Localhost".
+	// Must be set if type is "Localhost".
```

Comment:
this is a slight improvement but makes cherry-picking way messier... sorry to churn on this, but can you drop the doc change into into a separate PR we'll just merge to master after this one merges?

Reply:
https://github.com/kubernetes/kubernetes/pull/117505